src/modules/meteor/index.js

```diff
@@ -50,12 +50,12 @@ export function setup(api) {
     if (config.ssl.upload !== false) {
       list.copy('Copying SSL Certificate Bundle', {
         src: resolvePath(basePath, config.ssl.crt),
-        dest: '/opt/' + config.name + '/config/bundle.crt'
+        dest: '/opt/' + config.name + '/config/certs/bundle.crt'
```

Comment:
I was trying to make both setups consistent with each other. I'll modify it to be unbreaking.